src/main/java/pixelperfect/Game.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
    */
   @Override
   public void simpleInitApp() {
-    spaceship = new Spaceship();
+		spaceship = Spaceship.getInstance();
```

Comment:
I envisioned Spaceship to be a sort of Game class. There can only exist one spaceship in a game. I can only think of the situation where you would start a new game after a finished game where you would have to set the old spaceship to be renewed.